.travis.yml

```diff
@@ -6,134 +6,12 @@ language: rust
-    # "super old glibc" to ensure that it runs on as many platforms as possible.
     - os: linux
-      env: TARGET=i686-unknown-linux-gnu
-           DOCKER=alexcrichton/rust-slave-dist:2015-10-20b
-      if: branch != master
-    - os: linux
-      env: TARGET=x86_64-unknown-linux-gnu
-           DOCKER=alexcrichton/rust-slave-dist:2015-10-20b
-           NO_ADD=1
-
-    # Cross builds happen in the `rust-slave-linux-cross` image to ensure that
-    # we use the right cross compilers for these targets. That image should
-    # bundle all the gcc cross compilers to enable us to build OpenSSL
-    - os: linux
-      env: TARGET=arm-unknown-linux-gnueabi
-           DOCKER=alexcrichton/rust-slave-linux-cross:2016-11-11
-           SKIP_TESTS=1
-      if: branch != master
-    - os: linux
-      env: TARGET=arm-unknown-linux-gnueabihf
-           DOCKER=alexcrichton/rust-slave-linux-cross:2016-11-11
-           SKIP_TESTS=1
-      if: branch != master
-    - os: linux
-      env: TARGET=armv7-unknown-linux-gnueabihf
-           DOCKER=alexcrichton/rust-slave-linux-cross:2016-11-11
-           SKIP_TESTS=1
-      if: branch != master
-    - os: linux
-      env: TARGET=aarch64-unknown-linux-gnu
-           DOCKER=alexcrichton/rust-slave-linux-cross:2016-11-11
-           SKIP_TESTS=1
-      if: branch != master
-    - os: linux
-      env: TARGET=x86_64-unknown-freebsd
-           DOCKER=alexcrichton/rust-slave-linux-cross:2016-11-11
-           SKIP_TESTS=1
-      if: branch != master
-    - os: linux
-      env: TARGET=x86_64-unknown-netbsd
-           DOCKER=alexcrichton/rust-slave-linux-cross:2016-11-11
-           SKIP_TESTS=1
-      if: branch != master
-    - os: linux
-      env: TARGET=powerpc-unknown-linux-gnu
-           DOCKER=powerpc-unknown-linux-gnu
-           SKIP_TESTS=1
-      if: branch != master
-    - os: linux
-      env: TARGET=powerpc64-unknown-linux-gnu
-           DOCKER=powerpc64-unknown-linux-gnu
-           SKIP_TESTS=1
-      if: branch != master
-    - os: linux
-      env: TARGET=powerpc64le-unknown-linux-gnu
-           DOCKER=powerpc64le-unknown-linux-gnu
-           SKIP_TESTS=1
-      if: branch != master
-    - os: linux
-      env: TARGET=mips-unknown-linux-gnu
-           DOCKER=mips-unknown-linux-gnu
-           SKIP_TESTS=1
-      if: branch != master
-    - os: linux
-      env: TARGET=mipsel-unknown-linux-gnu
-           DOCKER=mipsel-unknown-linux-gnu
-           SKIP_TESTS=1
-      if: branch != master
-    - os: linux
-      env: TARGET=mips64-unknown-linux-gnuabi64
-           DOCKER=mips64-unknown-linux-gnuabi64
-           SKIP_TESTS=1
-      if: branch != master
-    - os: linux
-      env: TARGET=mips64el-unknown-linux-gnuabi64
-           DOCKER=mips64el-unknown-linux-gnuabi64
-           SKIP_TESTS=1
-      if: branch != master
-    - os: linux
-      env: TARGET=s390x-unknown-linux-gnu
-           DOCKER=s390x-unknown-linux-gnu
```

Comment:
Why remove all the other builds?

Reply:
Just to go easy on Travis resources and get a result faster while the build is failing there.
That commit will be removed once i figured that out.